src/lib/data.ts

```diff
@@ -113,7 +113,8 @@ export enum Replication {
 
 const defaults: Partial<DataSchema> = {};
 const store: Partial<DataSchema> = {};
-const event_emitter = new TypedEventEmitter<Events>();
+
+export const event_emitter = new TypedEventEmitter<Events>();
```

Comment:
But remote_data_sync_complete is an event, not a stored-data-value...

Reply:
(I thought whether you mean "why don't you watch for the thing that's being synced?"   The answer is because you may need to _avoid_ taking actions if the synced thing is a flag ... for example, if the person turned off help, you don't want to trigger a help flow ... that sort of thing).

Reply:
> But remote_data_sync_complete is an event, not a stored-data-value...

Ohhh I see.  I think until recently, that `event_emitter` only had events that corresponded to a key.

I actually agree that `watch()` would be a "semantic perversion", but I kind of disagree that `event_emitter` should be used for events that aren't directly attached to the setter.. Mild chance of key collision, and breaks the type schema 😅

Reply:
Like couldn't HelpFlows just watch the socket directly? Why does it need to use data's private event_emitter?

```
socket.on("remote_storage/sync_complete", updateHelpState);
```

Reply:
Then HelpFlows needs to know how the whole data socket thing works.   That seems like breaking encapsulation ...